graphql_compiler/compiler/schema_gen/exceptions.py

```diff
@@ -0,0 +1,33 @@
+class InvalidClassError(Exception):
```

Comment:
Maybe add a base class for these exceptions?

Reply:
But why?

Reply:
I talked about this with Predrag and for me it doesn't seem that it adds much value. I imagined having a SchemaGenerationError base case, but defacto imports InvalidClassError and uses it for stuff that isn't related to schema. I also think that we might want to use these errors for things other than schema generation in the future. The ModelBaseException base case doesn't seem to constrict the errors in a meaningful way, while still forcing the reader to read it's somewhat vague documentation.

Reply:
After thinking about it and having discussed it earlier with Predrag. I don't think it's a good idea. I don't think that the base class can constrict the errors in an informative manner and I think it will just waste reader's time so that we can get a nice sense of hierarchy.